docs/getting-started.md

```diff
@@ -44,10 +44,11 @@ repository](https://github.com/kubernetes/kubernetes) will automatically format
 and mount the requested Local SSDs, so you can deploy the provisioner with the
 pre-generated deployment spec and skip to [step
 4](#step-4-create-local-persistent-volume-claim), unless you want to customize
-the provisioner spec or storage classes.
+the provisioner spec or storage classes. We need to download the release from https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG, the latest release at the time of writing is v1.23.5.
```

Comment:
we can ask people to download release tar file, but I suggest don't write specific version in docs.

Reply:
It's intended to show as a demonstration with the latest build. Could you please kindly provide the suggested change from your side?